test/groovy/WhitesourceExecuteScanTest.groovy

```diff
@@ -100,6 +101,53 @@ class WhitesourceExecuteScanTest extends BasePiperTest {
         nullScript.commonPipelineEnvironment.configuration['steps']['whitesourceExecuteScan']['userTokenCredentialsId'] = 'ID-123456789'
     }
 
+    @Test
+    void testDockerFromCustomStepConfiguration() {
+
+        def expectedImage = 'image:test'
+        def expectedEnvVars = ['env1': 'value1', 'env2': 'value2']
+        def expectedOptions = '--opt1=val1 --opt2=val2 --opt3'
+        def expectedWorkspace = '/path/to/workspace'
+        
+        helper.registerAllowedMethod("readProperties", [Map], {
+            def result = new Properties()
+            result.putAll([
+                "apiKey": "b39d1328-52e2-42e3-98f0-932709daf3f0",
```

Comment:
Is the apiKey a real value or is this some kind of a fake?

Reply:
all i know is, that the key is expected. so, i copied from a similar test case.